### PR TITLE
Validation fixes

### PR DESCRIPTION
**Pull request by chrisms86**
## Category

- [x] Content fix
- [ ] New article

## Related issues

- fixes # 30

## What's in this Pull Request?

Validation fixes including converting tables to Markdown

### CODE REVIEW

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[a23c3a7](https://github.com/chrisms86/sp-dev-docs/commits/a23c3a794bbf3b7835691be72d591dbe6eb5a1de)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-a1dbbac0b1f2f2775b4b95d5a3fc47fd-ea6c7212)

<a id="a1dbbac0b1f2f2775b4b95d5a3fc47fd-ea6c7212"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206101142343945-8119/BuildReport?accessString=567714361df3751a21f3dda63859c58f983f8e9cf1c182273f5ba007ba9177f4).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[85af1a6](https://github.com/chrisms86/sp-dev-docs/commits/85af1a6eaa97b00df1c7d36b20ae1fb857929208)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-a1dbbac0b1f2f2775b4b95d5a3fc47fd-93c135b8)

<a id="a1dbbac0b1f2f2775b4b95d5a3fc47fd-93c135b8"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206101152204397-8119/BuildReport?accessString=82be9cfe1ee4361aef2071b5b4a3d5af99bf09670999fcf39500a86cdea479fa).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[69629f3](https://github.com/chrisms86/sp-dev-docs/commits/69629f3ae5c54f8a10372a7491328deaa89e4c6f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/listtemplate-element-list-template?branch=pr-en-us-8119) | [Details](#user-content-b006e005fbfea4a000fa59faf222c154-7d082954)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-a1dbbac0b1f2f2775b4b95d5a3fc47fd-7d082954)

<a id="b006e005fbfea4a000fa59faf222c154-7d082954"></a>
### [docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="a1dbbac0b1f2f2775b4b95d5a3fc47fd-7d082954"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131144225071-8119/BuildReport?accessString=f9b505dcc07a57320846491b14de7408b9926c4e48b25b0080276103fb27d0f2).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[a513d3a](https://github.com/chrisms86/sp-dev-docs/commits/a513d3acb8bc18776e24f00b37ea7b279d4cae73)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/listtemplate-element-list-template?branch=pr-en-us-8119) | [Details](#user-content-b006e005fbfea4a000fa59faf222c154-e8e0e242)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema?branch=pr-en-us-8119) | [Details](#user-content-13827aef853e387e34d179cabffb72ba-e8e0e242)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-a1dbbac0b1f2f2775b4b95d5a3fc47fd-e8e0e242)

<a id="b006e005fbfea4a000fa59faf222c154-e8e0e242"></a>
### [docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="13827aef853e387e34d179cabffb72ba-e8e0e242"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="a1dbbac0b1f2f2775b4b95d5a3fc47fd-e8e0e242"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131148491252-8119/BuildReport?accessString=6ba4117aa74d2c7cd9aa13867d85515271f8b478fd4ada10874f36b6c23aebfd).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[34a9304](https://github.com/chrisms86/sp-dev-docs/commits/34a93049b57c45af8c971a1d338d017b2dae6279)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/listtemplate-element-list-template?branch=pr-en-us-8119) | [Details](#user-content-b006e005fbfea4a000fa59faf222c154-d7e36ea5)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema?branch=pr-en-us-8119) | [Details](#user-content-13827aef853e387e34d179cabffb72ba-d7e36ea5)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-a1dbbac0b1f2f2775b4b95d5a3fc47fd-d7e36ea5)
[docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-15d778ea50111f3d3bf45495a0682e73-d7e36ea5)

<a id="b006e005fbfea4a000fa59faf222c154-d7e36ea5"></a>
### [docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="13827aef853e387e34d179cabffb72ba-d7e36ea5"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="a1dbbac0b1f2f2775b4b95d5a3fc47fd-d7e36ea5"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="15d778ea50111f3d3bf45495a0682e73-d7e36ea5"></a>
### [docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131154103047-8119/BuildReport?accessString=bed3572e5370ff34c115f43f713e7858b577cc6dcec9c08ca397b8b5928cf2e4).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by VesaJuvonen** [comment]
Docs Build status updates of commit _[c94f324](https://github.com/chrisms86/sp-dev-docs/commits/c94f3248894842579e448318a21eba985cd4e41c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/listtemplate-element-list-template?branch=pr-en-us-8119) | [Details](#user-content-b006e005fbfea4a000fa59faf222c154-d5109ad3)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema?branch=pr-en-us-8119) | [Details](#user-content-13827aef853e387e34d179cabffb72ba-d5109ad3)
[docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-a1dbbac0b1f2f2775b4b95d5a3fc47fd-d5109ad3)
[docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadata-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadata-schema?branch=pr-en-us-8119) | [Details](#user-content-435094aa01a18a7bd76d02151d8db8ca-d5109ad3)
[docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/sharepoint/dev/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema?branch=pr-en-us-8119) | [Details](#user-content-15d778ea50111f3d3bf45495a0682e73-d5109ad3)

<a id="b006e005fbfea4a000fa59faf222c154-d5109ad3"></a>
### [docs/schema/listtemplate-element-list-template.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/listtemplate-element-list-template.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="13827aef853e387e34d179cabffb72ba-d5109ad3"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="a1dbbac0b1f2f2775b4b95d5a3fc47fd-d5109ad3"></a>
### [docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsystem-element-in-lobsystems-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="435094aa01a18a7bd76d02151d8db8ca-d5109ad3"></a>
### [docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadata-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadata-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

<a id="15d778ea50111f3d3bf45495a0682e73-d5109ad3"></a>
### [docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/schema/lobsysteminstance-element-in-lobsysteminstances-bdcmetadataresource-schema.md)
 - **Line 2, Column 1**: **[Warning: ms-prod-and-service - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-prod-and-service?branch=main)]** `````Only one of the following attributes can exist: 'ms.prod', 'ms.service'. Use ms.prod for on-premise products, or ms.service for cloud services.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/8c4da01d-e430-7fe7-0bea-2c4100cce9e8/202206131156511362-8119/BuildReport?accessString=005ae88aa0979f795eaae64bfb461551bdfce8d43989a02d03cc06377d1ea695).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/8c4da01d-e430-7fe7-0bea-2c4100cce9e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/schema/listtemplate-element-list-template.md [hunk 1]
```diff
@@ -69,229 +70,59 @@ The following sections describe attributes, child elements, and parent elements.
...
-<col width="100%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td align="left"><p><a href="elements-element-list-template.md">Elements</a></p></td>
-</tr>
-</tbody>
-</table>
-
+[Elements](elements-element-list-template.md)
```
Comment (RedZephyr13) [inline docs/schema/listtemplate-element-list-template.md:125]: We've been directed to keep Markdown tables as tables. You can use "Element" or "Elements" as the header, or if you find a more specific header that fits, use that.

**Comment by chrisms86** [comment]
**Global effort to fix build validation errors**

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Review by andrewconnell — APPROVED** [review]

**Pull request merged**